biography.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="eng">
+  <head>
+    <link rel="stylesheet" href="indexStyles.css" />
+    <link rel="stylesheet" href="biographyStyles.css" />
+  </head>
+  <body>
+    <!--Header Section-->
+    <section class="header">
+      <div class="link-container">
+        <a class="header-link" href="index.html">Home</a>
+      </div>
+      <div class="link-container">
+        <a class="header-link" href="biography.html" id="biography">Biography</a>
+      </div>
+      <div class="link-container">
+        <a class="header-link" href="experience.html">Experience</a>
+      </div>
+      <div class="link-container">
+        <a class="header-link" href="portfolio.html">Portfolio</a>
+      </div>
+      <div class="underliner">
+        <div class="logo">Portfol.io</div>
+      </div>
+    </section>
+    <!--Header Section-->
+
+    <!--Profile Section-->
+    <div class="main">
+      <section class="profile">
+        <img
+          src="https://avatars.githubusercontent.com/u/109642366?v=4"
+          alt="profile image"
+          class="profile-image"
+        />
+        <p class="about-me" id="signature">Eric Roberson</p>
+        <div class="social-links">
+          <p class="about-me">LinkedIn</p>
+          <p class="about-me">GitHub</p>
+        </div>
+      </section>
+      <!--Profile Section-->
+
+       <!--Everything Else-->
+       <section class="home-section">
+        <div class="top-container">
+        <div class="top-section">
+          <!-- <img
+            id="theme-image"
+            src="https://media2.giphy.com/media/zGcAFoOCN8S3jzoIuv/giphy.gif?cid=ecf05e47gz68d6jsk7bmlhwcfmjrs3rzvvhdq93eat62wcyu&ep=v1_gifs_search&rid=giphy.gif&ct=g"
+          /> -->
+          <h2>more about me</h2>
+        </div>
+        <div class="stories-section">
+          <img class ="story-pic" src="https://images.unsplash.com/photo-1449034446853-66c86144b0ad?auto=format&fit=crop&q=80&w=2070&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D" alt=""/>
```

Comment:
for accessibility purposes, make sure your alt attributes are filled in with a description of each image